session.rst

```diff
@@ -1,15 +1,276 @@
+        $flashes->add('error', 'Another error');
+
+After processing the request, the controller sets a flash message in the session
+and then redirects. The message key (``notice`` in this example) can be anything:
+you'll use this key to retrieve the message.
+
+In the template of the next page (or even better, in your base layout template),
+read any flash messages from the session using the ``flashes()`` method provided
+by the :ref:`Twig global app variable <twig-app-variable>`:
+
+.. configuration-block::
+
+    .. code-block:: html+twig
+
+        {# templates/base.html.twig #}
+
+        {# read and display just one flash message type #}
+        {% for message in app.flashes('notice') %}
+            <div class="flash-notice">
+                {{ message }}
+            </div>
+        {% endfor %}
+
+        {# read and display several types of flash messages #}
+        {% for label, messages in app.flashes(['success', 'warning']) %}
+            {% for message in messages %}
+                <div class="flash-{{ label }}">
+                    {{ message }}
+                </div>
+            {% endfor %}
+        {% endfor %}
+
+        {# read and display all flash messages #}
+        {% for label, messages in app.flashes %}
+            {% for message in messages %}
+                <div class="flash-{{ label }}">
+                    {{ message }}
+                </div>
+            {% endfor %}
+        {% endfor %}
+
+    .. code-block:: php-standalone
+
+        // display warnings
+        foreach ($session->getFlashBag()->get('warning', []) as $message) {
+            echo '<div class="flash-warning">'.$message.'</div>';
+        }
+
+        // display errors
+        foreach ($session->getFlashBag()->get('error', []) as $message) {
+            echo '<div class="flash-error">'.$message.'</div>';
+        }
+
+        // display all flashes at once
+        foreach ($session->getFlashBag()->all() as $type => $messages) {
+            foreach ($messages as $message) {
+                echo '<div class="flash-'.$type.'">'.$message.'</div>';
+            }
+        }
+
+It's common to use ``notice``, ``warning`` and ``error`` as the keys of the
+different types of flash messages, but you can use any key that fits your
+needs.
+
+.. tip::
+
+    You can use the
+    :method:`Symfony\\Component\\HttpFoundation\\Session\\Flash\\FlashBagInterface::peek`
+    method instead to retrieve the message while keeping it in the bag.
 
 Configuration
 -------------
 
-Sessions are provided by the `HttpFoundation component`_, which is included in
-all Symfony applications, no matter how you installed it. Before using the
-sessions, check their default configuration:
+In the Symfony framework, sessions are enabled by default. Session storage and
+other configuration can be controlled under the :ref:`framework.session
+configuration <config-framework-session>` in
+``config/packages/framework.yaml``:
 
 .. configuration-block::
```

Comment:
Let's add a `.. code-block:: php-standalone` here as well (we do that in the HttpClient docs too), applies to other configuration examples as well.

Reply:
Done. I've done it for some other sections as well, but not for the DB configurations.